src/options.js

```diff
@@ -11,6 +11,42 @@ if (BUILD_COMPAT_REACT_NATIVE) {
   }
 }
 
+/**
```

Comment:
Where does all these texts come from? Did you just write these?

Reply:
Most were in the [sdk dev docs](https://developers.amplitude.com/docs/advanced-settings#setting-configuration-options-javascript). A couple I wrote

Reply:
Not seeing the `logAttributionCapturedEvent` property here.  Would be happy to write the details for the option, just wondering if it was left out on purpose?

Reply:
Yeah would be great if you could help add it!